orderbook/src/event_updater.rs

```diff
@@ -18,7 +18,7 @@ pub struct EventUpdater(Mutex<EventHandler<DynWeb3, GPv2SettlementContract, Data
 #[async_trait::async_trait]
 impl EventStoring<ContractEvent> for Database {
     async fn replace_events(
-        &self,
+        &mut self,
```

Comment:
Now that I see it (and we talked about it in the office), this change - while not a lot of lines - could have gone separate to keep the PR more focussed.